frontend/src/routes/Dataset/Dataset.js

```diff
@@ -23,6 +25,14 @@ function Dataset() {
   const datasetName = dataset?.title || ''
   const searchPlaceholder = datasetConfig?.searchPlaceholder || ''
 
+  const [selectedSearchResult, setSelectedSearchResult] = useState(null)
+  const [highlightedSearchResult, setHighlightedSearchResult] = useState(null)
+
+  const handleSearchChange = useCallback(e => {
+    // TODO: 'small-business' -> dataset?.id
```

Comment:
@justinanderson shall this endpoint be changed to "small-business-support"? **datasetId** has that value in this case